core/src/main/kotlin/live/LiveKordEntity.kt

```diff
@@ -27,11 +26,10 @@ public interface LiveKordEntity : KordEntity, CoroutineScope {
 
 @KordPreview
 public abstract class AbstractLiveKordEntity(
-    override val kord: Kord,
+    final override val kord: Kord,
```

Comment:
reason:
fixes warning "Accessing non-final property `kord` in constructor" in final `events` property